be/src/vec/functions/round.h

```diff
@@ -217,75 +230,30 @@ class BaseFloatRoundingComputation<Float64> {
     static VectorType divide(VectorType val, VectorType scale) { return _mm_div_pd(val, scale); }
     template <RoundingMode mode>
     static VectorType apply(VectorType val) {
-        return _mm_round_pd(val, int(mode));
+        if constexpr (mode != RoundingMode::Round) {
+            return _mm_round_pd(val, int(mode));
+        }
+        VectorType result;
+        Float64 input_values[2];
+        _mm_storeu_pd(input_values, val);
+        if constexpr (tie_breaking_mode == TieBreakingMode::Bankers) {
+            result = _mm_set_pd(std::nearbyint(input_values[1]), std::nearbyintf(input_values[0]));
```

Comment:
`_mm_set_pd` is a sequence instruction. since you have put the input values in contiguous places(`input_values`), don't use this, but use `_mm_loadu_pd`